FF1Lib/FF1Rom.cs

```diff
@@ -86,17 +86,17 @@ public void Randomize(Blob seed, Flags flags)
 			var rng = new MT19337(BitConverter.ToUInt32(seed, 0));
 
 			UpgradeToMMC3();
-			EasterEggs();
```

Comment:
There are a lot of unnecessary whitespace changes in this file... here and several other places.